build.sbt

```diff
@@ -22,13 +26,46 @@ lazy val cross = crossProject.in(file(".")).
     developers := List(Developer("benhutchison", "Ben Hutchison", "brhutchison@gmail.com", url = url("https://github.com/benhutchison"))),
     scmInfo := Some(ScmInfo(url("https://github.com/typelevel/mouse"), "scm:git:https://github.com/typelevel/mouse.git")),
     scalacOptions ++= Seq("-feature", "-deprecation", "-language:implicitConversions"),
+    publishMavenStyle := true,
+    publishArtifact in Test := false,
+    pomIncludeRepository := { _ => false },
     publishTo := Some(
       if (isSnapshot.value)
         Opts.resolver.sonatypeSnapshots
       else
         Opts.resolver.sonatypeStaging
-      )
+      ),
+    pomExtra := (
+      <developers>
+        <developer>
+          <id>benhutchison</id>
+          <name>Ben Hutchison</name>
+          <url>http://benhutchison.wordpress.com</url>
+        </developer>
+      </developers>
+     ),
```

Comment:
Is `pomExtra` needed now that `developers` key (see line 22 above) is available?

Reply:
ha, maybe not, this must be something new sbt-sonatype.